src/ptvsd/server/api.py

```diff
@@ -13,10 +13,19 @@
 import ptvsd
 from ptvsd.common import log, options as common_opts
 from ptvsd.server import options as server_opts
+from ptvsd.common.compat import queue
 from _pydevd_bundle.pydevd_constants import get_global_debugger
 from pydevd_file_utils import get_abs_path_real_path_and_base_from_file
 
 
+_ADAPTER_START_TIMEOUT = 10
+_dap_listener = None
+
+
+class EnableAttachError(Exception):
+    pass
```

Comment:
No need for a custom exception type for this, since it's not really meant to be caught. `RuntimeError` with the appropriate message type will do the trick.